windows_change_test_configs.py

```diff
@@ -0,0 +1,75 @@
+#No header files detected
```

Comment:
?

Reply:
Is this script used anyway?

Reply:
set "changeConfigsCmd=windows_change_test_configs.py" in build_windows.bat